crates/core_simd/src/ops.rs

```diff
@@ -144,8 +144,8 @@ macro_rules! impl_op {
             impl<const LANES: usize> core::ops::Neg for crate::$type<LANES> {
                 type Output = Self;
                 fn neg(self) -> Self::Output {
-                    Self::splat(0) - self
-                }
+                            Self::splat(0) - self
+                    }
```

Comment:
Hm, this indentation change isn't terribly important, but it's odd. Did you run `cargo fmt`?

Reply:
Oooh, did not know that existed.
Thanks, will clean up.